backend/get_past_matches.js

```diff
@@ -0,0 +1,16 @@
+const sdk = require('api')('@developers-pandascore/v2#apvn9q1ml4sw7gei');
+import * as dotenv from 'dotenv'
+
+dotenv.config(({path:__dirname+'/.env'}))
+
+sdk.auth(process.env.APIKEY);
+
+sdk['get_matches_past']({
+    'filter[finished]': 'true',
```

Comment:
true, false and ints have to be quoted here?

Reply:
Both work as intended, removed quotes to be considered `bool` and not `string`
Done 👍🏻